evergreen/config.yml

```diff
@@ -149,11 +149,18 @@ functions:
               set_cmake_var generator_vars CMAKE_MAKE_PROGRAM PATH "${ninja|ninja}"
           fi
 
+          if [ -n "${cmake_generator_platform|}" ]; then
+            set_cmake_var realm_vars CMAKE_GENERATOR_PLATFORM STRING "${cmake_generator_platform}"
+          fi
+
           if [ -n "${curl_base|}" ]; then
               set_cmake_var curl_vars CURL_LIBRARY PATH "$(./evergreen/abspath.sh ${curl_base}/lib/curl.lib)"
               set_cmake_var curl_vars CURL_INCLUDE_DIR PATH "$(./evergreen/abspath.sh ${curl_base}/include)"
+              set_cmake_var baas_vars REALM_CURL_CACERTS PATH "$(./evergreen/abspath.sh "${curl_base}/bin/cacert.pem")"
           fi
 
+          set_cmake_var realm_vars REALM_NO_TESTS BOOL ${no_tests|Off}
```

Comment:
Is this a flag that will be used in the future for compile lib only tasks?

Reply:
Yes, I guess so. It's how we make sure that compile-only builders are actually compile only in Jenkins.